src/main/java/org/prebid/server/bidder/tripleliftnative/TripleliftNativeBidder.java

```diff
@@ -57,7 +54,7 @@ public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest bidRequ
         final List<Imp> validImps = new ArrayList<>();
         for (Imp imp : bidRequest.getImp()) {
             try {
-                validImps.add(modifyImp(imp));
+                validImps.add(modifyImp(imp, bidRequest));
```

Comment:
You can extract `publisher.domain` first (so you don't repeat the same action on each imp).
Also, please, extract `final TripleliftNativeExtImp impExt = parseExtImp(imp);` from `modifyImp` method to make it more in "style" with other bidders.